File: Java/source/src/main/java/com/github/yitter/contract/ISnowWorker.java
/*
 * 版权属于：yitter(yitter@126.com)
 * 开源地址：https://gitee.com/yitter/idgenerator
 */
package com.github.yitter.contract;

public interface ISnowWorker {
    long nextId() throws IdGeneratorException;
}

File: Go/source/idgen/ISnowWorker.go
/*
 * 版权属于：yitter(yitter@126.com)
 * 代码编辑：guoyahao
 * 代码修订：yitter
 * 开源地址：https://github.com/yitter/idgenerator
 */
package idgen

type ISnowWorker interface {
	NextId() int64
}
